assets/src/application/ui/input/InlineEditInput/style.scss

```diff
@@ -12,6 +12,12 @@
 			height: 2.625rem; // 42px
 			width: 2.625rem; // 42px
 
+			.ee-infinity-sign__inner {
+				font-weight: 600;
+				position: relative;
+				top: -3px;
```

Comment:
this is for vertical alignment?

Reply:
ya when you hover over the infinity sign, the blue border appears since it is editable, but the text is too low within the icon button square. Tried everything else I could to center it, but nothing else worked for some strange reason.